packages/statemanager/src/interface.ts

```diff
@@ -11,41 +10,30 @@ export interface StorageDump {
 
 export type AccountFields = Partial<Pick<Account, 'nonce' | 'balance' | 'stateRoot' | 'codeHash'>>
 
-export interface StateManager {
-  copy(): StateManager
+export interface StateAccess {
```

Comment:
Is this differentiation between `StateAccess` and `StateManager` just to make things a bit cleaner or does this have practical implications?

Just asking for some context.

Reply:
I guess you have already found the `VmStateAccess` inheriting from this to guarantee proxing these calls to the `StateManager` :slightly_smiling_face: :+1: